prover/src/opcodes/integer_ops.rs

```diff
@@ -473,22 +613,30 @@ mod tests {
                 ;; Skip @4 to test a gap in vrom writes
                 SUB @5, @2, @3\n\
                 ADD @6, @2, @3\n\
+                MULU @8, @2, @3\n\
```

Comment:
Question: why not 'MULU @8, @2, @3\n\'? Is it because of the need for alignment? If so, what are the alignment requirements?

Reply:
You mean why not `MULU @7, @2, @3`? Yes, it's for alignment. Addresses are binary elements where + is XOR, which is why we want cell `i` of an object of size `32*n` and at base address `addr` to be accessible with `addr XOR i`.